blog/2022-08-04-Building interactive data applications in NeuroBlu with Plotly Dash.md

```diff
@@ -0,0 +1,89 @@
+---
```

Comment:
Could you please rename the file so it doesn't contain spaces in filename?
All the other blog posts use the convention of replacing spaces in filename with '-', so we should probably keep that convention.

Reply:
Apologies @jhrcek - I didn't see this comment notification! I replaced spaces with dashes

Reply:
Thank you